### ISSUE TITLE
compatibility with pytest-dependency

### ISSUE THREAD

**Issue by Mystorius** [jerry-git/pytest-split#75]
Hi,

first: thank you for the cool tool, it really helps with faster CI pipelines :)

My request: can you make the split algorithm take pytest dependencys into account? We have same test that depend on other test to run first. Currently pytest splits theses into different groups which will result in the tests failing.

**Comment by jerry-git** [comment]
Sorry, I think I misread initially. So, you are basically asking if pytest-split could be made compatible with https://pypi.org/project/pytest-dependency/?

**Comment by Mystorius** [comment]
Yes, was just about to post that we use [pytest-dependency](https://pypi.org/project/pytest-dependency) But I didn't know that pytest-split is compatible with fixtures already. Maybe it would be worth it to refactor our tests into fixtures then as pytest-dependency seems to be out of date anyways.

**Comment by jerry-git** [comment]
> Maybe it would be worth it to refactor our tests into fixtures

Great! I hope it works out nicely for you 🙂
